### DISCUSSION TITLE
[Resolved] Deployments down?

### DISCUSSION THREAD

**Discussion by samuelcole** [vercel/community#2591] [Bug Report]
I'm getting this error across all my apps, after the build finishes:

```
Build Completed in /vercel/output [5m]
--
Deploying outputs...
Error: Not Found
```

**Comment by carlosdarwed** [comment]
Same with me, started accusing me about the node version, and then it didn't deploy any of the commits that were made

**Reply by carlosdarwed** [reply]
yep, same

**Comment by ethndotsh** [comment]
I too am having this issue.

**Comment by swarnava** [comment]
Hi,

We have identified an issue and working on a fix.

**Comment by jfspencerAngel** [comment]
We are not able to deploy anything either, in a not found loop after clone
![not_found](https://github.com/vercel/community/assets/87774692/6758b905-7401-4dfb-8e80-3ae14a1cd7ba)

**Comment by swarnava** [comment]
Here is the latest update: https://www.vercel-status.com/incidents/nvftyfvh18s2

**Comment by swarnava** [comment]
A fix has been rolled out and we are processing queued build slowly.
